starlark/library.go

```diff
@@ -1889,6 +1891,22 @@ func string_partition(_ *Thread, b *Builtin, args Tuple, kwargs []Tuple) (Value,
 	return tuple, nil
 }
 
+// https://github.com/google/starlark-go/blob/master/doc/spec.md#string·removeprefix
+// https://github.com/google/starlark-go/blob/master/doc/spec.md#string·removesuffix
+func string_removefix(_ *Thread, b *Builtin, args Tuple, kwargs []Tuple) (Value, error) {
+	recv := string(b.Receiver().(String))
+	var fix string
+	if err := UnpackPositionalArgs(b.Name(), args, kwargs, 1, &fix); err != nil {
+		return nil, err
+	}
+	if b.name[6] == 'p' {
```

Comment:
As the function is short, it might be worth duplicating it instead of using `b.name` magic.

If you want to keep it like this, add a comment on the line

Reply:
Since it's easy to avoid the duplication of source and object code (240 insns -> 160), and the cost of the extra load and two cjmps is trivial overall, I'd prefer to keep it this way.

But you're right it could be clearer. Commented in the form of self-documenting code:
```
	if b.name[len("remove")] == 'p' {
```